### ISSUE TITLE
Error when building from source

### ISSUE THREAD

**Issue by neon64** [asmagill/hs._asm.undocumented.spaces#9]
I haven't got a chance to test it yet, but this module should be really useful for my upcoming tiling window manager built with hs!

Except when trying to build from source. I get this from `make install`: 

```
luac-5.3 -p init.lua && echo "Passed"
Passed
mkdir -p ~/.hammerspoon/hs/_asm/undocumented/chris spaces
install -m 0644 internal.so ~/.hammerspoon/hs/_asm/undocumented/chris spaces
install: /Users/chris/.hammerspoon/hs/_asm/undocumented/chris: Inappropriate file type or format
make: *** [install-objc] Error 71
```

Firstly I got an error about `luac-5.3` not being found (which perhaps should be added as a requirement in README.md) but that was easily fixed with `brew install lua53`.

Now I don't know how to diagnose the error:

`/Users/chris/.hammerspoon/hs/_asm/undocumented/chris: Inappropriate file type or format`

I thought perhaps it has something to do with the space in the path `~/.hammerspoon/hs/_asm/undocumented/chris spaces` because it would get interpreted by shell commands as two paths. Why would `chris spaces` be in the install path anyway?

Thanks if you could help.

**Comment by asmagill** [comment]
The Makefile determines the module's last name component by looking at the directory that it is in; at a guess I would say that it looks like you downloaded the files into a directory named "chris", so it's trying to install it as `hs._asm.undocumented.chris`.  The simple fix would be to create a new directory called "spaces" and then move the source into there and compile from the new sub-directory.  The long term fix is that I need to adjust the documentation to make this more clear and maybe add a flag to the Makefile so that the module's node name can be specified and only use the subdirectory name as a fallback. I won't have time  to address this any time soon, but I will leave this open as a reminder to get to it when I can.

**Comment by asmagill** [comment]
Fixed with 0b5321fc336f75488fb4bbb524677bb8291050bd
